sk homework done

sdkevorkian opened the pull request:
Comfort Level (1-5):
5
Completeness Level (1-5):
5
What did you think of this deliverable?
It was great for learning to manage large amounts of HTML and CSS,  I haven't worked with multiple pages before or this much. I did have a problem with getting the img gallery to vertically align correctly. was able to do it by wrapping the imgs in 3 divs of 2 imgs, but then on the media query it wouldn't line up in 2 columns of 3 imgs. how would i get the vertical alignment correct in this set up? also, is there a way using javascript later on to change the arrangement of child elements in parent elements? (for example, could I have my div set up as previously described, but then on smaller screens move the img elements between divs and delete 1 div?) I suppose we will see!!

sdkevorkian committed:
made html for blog and styled with css

sdkevorkian committed:
added background image to header, enhaned footer

sdkevorkian committed:
updated widths to max-width to allow for scaling

sdkevorkian committed:
added flexbox layout to main, made header nav fixed, included hover effect over header/footer nav

sdkevorkian committed:
made page more responsive, added forms, changed px to em where possibly

sdkevorkian committed:
minor adjustments, added compatibility prefixes

sdkevorkian committed:
added landing page, next step deal with contact form on landing page and make responsive

sdkevorkian committed:
adjusted forms on both pages, made most of page responsive. this is last save before i rehaul the img gallery

sdkevorkian committed:
got the gallery flexbox more or less working, some weird spacing still going on...

sdkevorkian committed:
added fun hover animations

sdkevorkian committed:
made generic column % classes to remove redundancy

psiie commented:
## CSS

Great use of comments & organization in your CSS. For now, lets not worry about the backwards compatibility of `-webkit-____: and -ms-____: `. These are for enabling 'not fully supported' features in other browsers. Seeing as we are using chrome and not launching these sites, we can ignore these for now. I just don't want these to be a hangup. :)

I'm really impressed with the `trippy` effects you made in CSS.

## Notes

I would consider making the type, buttons and inputs larger for those on mobile phones, as well as making the pictures be one column when the browser is this tiny. How would you accomplish this?

sdkevorkian committed:
added form validation and made font larger on mobile

sdkevorkian commented:
Thanks!! I actually used an autoprefixer so it was no trouble lol. I went and added the font-size of the body in my media query, I upped it to 20px, so the ems updated for mobile. the deliverable asked for two columns on mobile, but to change it to 1 i would change the max-width of the images to ~100%

not sure how pull requests work after making changes, so i'll put my comments for part 5 for now in here. 

Comfort Level (1-5):
5
Completeness Level (1-5):
5
What did you think of this deliverable? it was good - very useful to go back and add jquery to something we already made. It made me reconsider how I organized the page (for example, I went back and added divs in my HTML to house the <p> message with the input field, so it would be easier to use .siblings() ). 

what I ran into trouble was that I tried a bunch of different combinations of "this" in my initial query to select the input fields (I wanted to select only input fields of the submit button I clicked on, aka this) and just couldnt get them selected.
